src/globals.cpp

```diff
@@ -363,6 +365,11 @@ CTweak<CAmount> maxTxFee("wallet.maxTxFee",
     "transactions.",
     DEFAULT_TRANSACTION_MAXFEE);
 
+CTweakRef<int> maxTxFeePercent("wallet.maxTxFeePercent",
+    "Maximum total fees to use in a single wallet transaction or raw transaction as a percent of the total value"
+    "(sent and change) of the transaction.  Set to a negative number to turn off",
+    &MAX_FEE_PERCENT_OF_VALUE);
+
```

Comment:
why do we need this one, wouldn't things like DEFAULT_TRANSACTION_MAXFEE be enough?

Reply:
Back in Core days, I thought that this was important because it reflects what users really care about.  But its not perfect because its the full amount, not what's being spent.  Its pretty much not relevant in BCH due to our large block philosophy.  Since both you and ptschip don't like this so I'll remove it.

Reply:
Removed